distributed/http/scheduler/tests/test_scheduler_http.py

```diff
@@ -245,3 +246,51 @@ async def test_eventstream(c, s, a, b):
     )
     assert "websocket" in str(s.plugins).lower()
     ws_client.close()
+
+
+@gen_cluster(client=True, clean_kwargs={"threads": False})
+async def test_api(c, s, a, b):
+    http_client = AsyncHTTPClient()
+
+    response = await http_client.fetch(
+        "http://localhost:%d/api/v1" % s.http_server.port
+    )
+    assert response.code == 200
+    assert response.headers["Content-Type"] == "text/plain"
+
+    txt = response.body.decode("utf8")
+    assert txt == "API V1"
+
+
+@gen_cluster(client=True, clean_kwargs={"threads": False})
+async def test_retire_workers(c, s, a, b):
+    async with aiohttp.ClientSession() as session:
+        params = {"workers": [a.address, b.address]}
+        async with session.post(
+            "http://localhost:%d/api/v1/retire_workers" % s.http_server.port,
+            json=params,
+        ) as resp:
+            assert resp.status == 200
```

Comment:
We should check that the workers actually retire. And also that we get the expected JSON response.

Reply:
And that is has a JSON header.